terraform/deployments/apps/variables.tf

```diff
@@ -23,12 +23,12 @@ variable "mesh_domain" {
 # Environment
 #--------------------------------------------------------------
 
-variable "app_domain" {
+variable "external_app_domain" {
   type        = string
   description = "Example: staging.publishing.service.gov.uk"
```

Comment:
Is this example actually correct? I thought this one was like `test.govuk.digital`, and that we had another variable for the service domain?